app/src/main/java/com/framgia/forder/screen/domainmanagement/DomainManagementPresenter.java

```diff
@@ -1,10 +1,15 @@
 package com.framgia.forder.screen.domainmanagement;
 
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports